### DISCUSSION TITLE
DockerComposeCompositeService: Fetch logs from a finished container when

### DISCUSSION THREAD

**Discussion by cvicenti-apr** [mariotoffia/FluentDocker#310] [Q&A]
Hi.
Has anyone managed to fetch logs from containers that are no longer running when passing a docker-compose.yml file?
I am able to retrieve logs from the instances that are discoverable at that particular moment but I'd need to see how the other ones went after they completed their intended execution.
Thanks in advance!

**Comment by SanjaykumarSharma18** [comment]
Can you please share, how were you able to see the logs while the containers are running?
